esp-hal/Cargo.toml

```diff
@@ -1,19 +1,20 @@
 [package]
-name         = "esp-hal"
-version      = "0.15.0"
-edition      = "2021"
-rust-version = "1.76.0"
-description  = "Bare-metal HAL for Espressif devices"
-repository   = "https://github.com/esp-rs/esp-hal"
-license      = "MIT OR Apache-2.0"
-links        = "esp-hal"
+name          = "esp-hal"
+version       = "0.16.0"
+edition       = "2021"
+rust-version  = "1.76.0"
+description   = "Bare-metal HAL for Espressif devices"
+documentation = "https://docs.esp-rs.org/esp-hal/"
```

Comment:
Has the DNS been changed for this yet? Or maybe my DNS hasn't updated :D

Reply:
imo we should, but it might delay our release because of DNS propagation times :thinking:

Reply:
Hopefully it will take less than 24hours, and we can check with https://www.whatsmydns.net/#A/docs.esp-rs.org - so maybe we still just go for it and pencil in the release for tomorrow?

Reply:
Usually only takes an hour or so IME, but sure sounds good.